src/main/java/core/basesyntax/figure/ColorSupplier.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.figure;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    private static final String[] COLORS = {
+            "red",
+            "blue",
+            "green",
+            "yellow",
+            "black",
+            "white",
+            "purple",
+            "gray"
+    };
+
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
To avoid creating a new instance of Random every time the method is called, consider moving the `Random` instance to a class-level variable.

Reply:
Fix this comment